go.mod

```diff
@@ -9,9 +9,11 @@ require (
 	github.com/openshift/client-go v0.0.0-20221019143426-16aed247da5c
 	github.com/project-codeflare/multi-cluster-app-dispatcher v1.37.0
 	github.com/ray-project/kuberay/ray-operator v0.0.0-20231016183545-097828931d15
+	github.com/stretchr/testify v1.8.4
```

Comment:
What is testify used for?

Reply:
I missed running `go mod tidy `It's not needed.